install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -147,6 +147,7 @@ type WebAppConfig struct {
 }
 
 type WorkspaceDefaults struct {
+	// @deprecated use workspace.workspaceImage instead
```

Comment:
Can we remove these rather than deprecating them? AIUI, the experimental config section is only ever used by Gitpod SaaS, so we should be able to update our usages of these fields (if any) to their new home outside of experimental.